doc/Command-Reference.md

```diff
@@ -10713,6 +10713,35 @@ This command is used to disable syslog rate limit feature.
   config syslog rate-limit-feature disable database -n asci0
   ```
 
+**config syslog level**
+
+This command is used to configure log level for a given log identifier.
+
+- Usage:
+  ```
+  config syslog level -c <log_identifier> -l <log_level> --service [<service_name>] --program [<program_name>]
+
+  config syslog level -c <log_identifier> -l <log_level> --service [<service_name>] --pid [<process_id>]
+
+  config syslog level -c <log_identifier> -l <log_level> ---pid [<process_id>]
+  ```
+
+- Example:
+
+  ```
+  # Update the log level without refresh the configuration
+  config syslog level -c xcvrd -l DEBUG
+
+  # Update the log level and send SIGHUP to xcvrd running in PMON
+  config syslog level -c xcvrd -l DEBUG --service pmon --program xcvrd
```

Comment:
@Junchao-Mellanox I am not able to distinguish between the component name and the program name which is same here and confusing.

Reply:
Will rename component to log_identifier